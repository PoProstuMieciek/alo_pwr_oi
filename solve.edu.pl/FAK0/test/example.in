6
1
2
42
314
7999999
8000000
