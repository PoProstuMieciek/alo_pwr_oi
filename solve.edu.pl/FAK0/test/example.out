1 
2 
2 3 7 
2 157 
7 199 5743 
2 2 2 2 2 2 2 2 2 5 5 5 5 5 5 
